src/examples/live-data-basic/example-live-data-basic.component.ts

```diff
@@ -13,24 +13,38 @@ export class ExampleLiveDataBasicComponent implements OnDestroy {
 	static id = 'live-data-basic';
 
 	rows: Quote[];
```

Comment:
this is example will be in documentation and code will be visible to the users, please make sure that code is simple enough so users can understand it from first glance